WordPress/Classes/Services/AccountService.m

```diff
@@ -242,6 +258,44 @@ - (NSUInteger)numberOfAccounts
     return count;
 }
 
+/**
+ Returns all accounts currently existing in core data.
+
+ @return An array of WPAccounts.
+ */
+- (NSArray<WPAccount *> *)allAccounts
+{
+    NSFetchRequest *fetchRequest = [NSFetchRequest fetchRequestWithEntityName:@"Account"];
+    NSError *error = nil;
+    NSArray *fetchedObjects = [self.managedObjectContext executeFetchRequest:fetchRequest error:&error];
+    if (error) {
+        return @[];
+    }
+    return fetchedObjects;
+}
+
+/**
+ Checks an account to see if it is just used to connect to Jetpack.
+
+ @param account The account to inspect.
+ @return True if used only for a Jetpack connection.
+ */
+- (BOOL)isJetpackAccount:(WPAccount *)account
```

Comment:
The logic seems good, but I think the naming is confusing. Perhaps `accountHasOnlyJetpackBlogs`? The current name would imply to me that this would return true if the account had any Jetpack blogs, regardless of other conditions.